test/integration/testpaths/testpath_disable_enable_zone.py

```diff
@@ -0,0 +1,492 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Move the file to component/maint/ folder so that it runs serially and not in parallel with other test suites.